src/components/Navbar/index.jsx

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 
 import { ButtonPrimary, ButtonSecondary } from '../Buttons';
 
 import { Brand, Header, HeaderStyles, HeaderStylesRight } from './navbar.style';
 
-export const Navbar = () => (
-  <HeaderStyles>
-    <Header>
-      <Brand>RECORD STORE</Brand>
-      <HeaderStylesRight>
-        <ButtonSecondary type='button'>Login</ButtonSecondary>
-        <ButtonPrimary type='button'>Sign up</ButtonPrimary>
-      </HeaderStylesRight>
-    </Header>
-  </HeaderStyles>
-);
+export const Navbar = () => {
+  const history = useHistory();
+
+  const handleGoToSignUp = () => history.push('/signup');
```

Comment:
Why are you using history to go to the signup page.  
Why not Link?

Reply:
To avoid having to style the `a` tags, they can be super annoying because of a11y issues.